samza-core/src/test/java/org/apache/samza/zk/TestZkUtils.java

```diff
@@ -89,11 +91,11 @@ public static void teardown() {
 
   @Test
   public void testRegisterProcessorId() {
-    String assignedPath = zkUtils.registerProcessorAndGetId("0.0.0.0");
+    String assignedPath = zkUtils.registerProcessorAndGetId("0.0.0.0 1");
```

Comment:
i guess we can do that.

Not sure who is using hostname, it was added by someone else.